protected/gatk-tools-protected/src/main/java/org/broadinstitute/gatk/tools/walkers/variantutils/CombineGVCFs.java

```diff
@@ -264,7 +294,8 @@ public OverallState reduce(final PositionalState startingStates, final OverallSt
      */
     private boolean breakBand(final GenomeLoc loc) {
         return USE_BP_RESOLUTION ||
-                (loc != null && multipleAtWhichToBreakBands > 0 && (loc.getStart()+1) % multipleAtWhichToBreakBands == 0);  // add +1 to the loc because we want to break BEFORE this base
+                (loc != null && multipleAtWhichToBreakBands > 0 && (loc.getStart()+1) % multipleAtWhichToBreakBands == 0) || // add +1 to the loc because we want to break BEFORE this base
+                (loc != null && breakBandsAt.contains(loc.getContig()+":"+(loc.getStart()+1)));
```

Comment:
String creation and comparisons are very (very) slow.  This should use the GenomeLoc class instead.